Review/week-1/scope_review.md

```diff
@@ -0,0 +1,62 @@
+## Lessons and Exercises
+- [Scope](https://github.com/Ada-Developers-Academy/textbook-curriculum/blob/master/01-ruby-fundamentals/scope.md)
+
+## Warm-up Exercise
+
+Review the code below. What do you notice? What do you wonder?
+
+### Check for understanding
+1. What is the method name?
+2. When do we call the method?
+3. What are the method parameters?
+4. What is the method's return value? What is the data type of this return value?
+5. What happens if we uncomment #1?
+6. What happens if we uncomment #2?
+7. What is the output of #3?
+8. What is the output of #4?
+
+```ruby
+
+# find the equation of a line given two points
+def equation(x1, y1, x2, y2)
```

Comment:
i think method names are much clearer when they're a _verb_ ish, or action. Could we rename this to something like `find_equation` or `find_line_equation`?

Reply:
I'll make this change.